intelmq/bin/intelmqctl.py

```diff
@@ -510,26 +522,36 @@ def __init__(self, interactive: bool = False, return_type: str = "python", quiet
             parser_start = subparsers.add_parser('start', help='Start a bot or botnet')
             parser_start.add_argument('bot_id', nargs='?',
                                       choices=self.runtime_configuration.keys())
+            parser_start.add_argument('--group', help='Start a group of bots',
+                                     choices=BOT_GROUP.keys())
```

Comment:
As None is also in `BOT_GROUP` this leads to the following output:
```
intelmqctl stop: error: argument --group: invalid choice: 'foobar' (choose from 'experts', 'outputs', 'parsers', 'collectors', None)
```
None is no valid input (and can't be an input)

Reply:
agree